docs/topics/mta-web-seeding-instances.adoc

```diff
@@ -26,7 +26,7 @@ Stakeholders:
 
 Tags:
 
-. Create tag types
+. Create categories
```

Comment:
same here, can you please replace with **Tag categories** ?

Reply:
Done